CRM/Prospect/Setup/AddProspectCategoryCgExtendsValue.php

```diff
@@ -23,12 +23,36 @@ public function apply() {
     civicrm_api3('OptionValue', 'create', [
       'option_group_id' => 'cg_extend_objects',
       'name' => 'civicrm_case',
-      'label' => $prospectCategoryLabel,
+      'label' => self::PROSPECTS_CATEGORY_LABEL,
       'value' => 'prospecting',
       'description' => 'CRM_Prospect_Helper_CaseTypeCategory::getProspectCaseTypes;',
       'is_active' => TRUE,
       'is_reserved' => TRUE,
     ]);
   }
 
+  /**
+   * Enables/Disables Prospects option values
+   *
+   * @param $newStatus
```

Comment:
Missing parameter comment.

You need to run npm install in the prospect extension root so that the php linter is added to git pre-commit hook. That way you get notified of things like this before you commit.